src/components/WorkflowForms/NewClientSignup/GatherInfo/GatherInfo.jsx

```
@@ -1,180 +1,534 @@
+      }
+      
+      const dataToSend = {
+        id: caseId,
+        policeReport: policeReportData,
+        additionalPassengers: values.additionalPassengers,
+        medicalProviders: medicalProviderIds
+      };
+
+      const updateResult = await api.update_whole_case(dataToSend);
+      
+      if (updateResult && updateResult.success) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)